### ISSUE TITLE
AsciiDoc: Consider switching to a different dependency for syntax highlighting

### ISSUE THREAD

**Issue by hertg** [sharkdp/bat#1290]
The repository which is used for syntax highlighting of AsciiDoc seems to be unmaintained (see [asciidoctor/sublimetext-asciidoc](https://github.com/asciidoctor/sublimetext-asciidoc.git)). There are issues and PRs unresolved / unanswered for 4 years and longer.

There is a minor issue with the syntax highlighting that was found while implementing AsciiDoc tests in PR #1229. The bug [has been reported](https://github.com/asciidoctor/sublimetext-asciidoc/issues/20) to the library mentioned above, but probably won't get fixed.

Consider switching to a different dependency for AsciiDoc syntax highlighting. Here are some more up-to-date libraries I've found that have AsciiDoc highlighting, but I don't know whether they could be used inside bat or not, as they are mostly JS libraries:

- [asciidoctor/asciidoctor-vscode](https://github.com/asciidoctor/asciidoctor-vscode)
  The vscode plugin for AsciiDoc
- [asciidoctor/asciidoctor-intellij-plugin](https://github.com/asciidoctor/asciidoctor-intellij-plugin)
  The IntelliJ plugin for AsciiDoc
- [PrismJS/prism](https://github.com/PrismJS/prism)
  A Javascript library that supports AsciiDoc highlighting and is up-to-date.
  I can confirm that the issue does not exist in this library:
  ![image](https://user-images.githubusercontent.com/3533648/95433415-2fdd5480-0950-11eb-8b29-a711bc6dab4f.png)
- [highlightjs/highlight.js](https://github.com/highlightjs/highlight.js)
  Another Javascript library that supports AsciiDoc highlighting.
  This library also has a similar bug, but the project is still maintained.
  ![image](https://user-images.githubusercontent.com/3533648/95433691-95314580-0950-11eb-9c6f-0f7a2de283fb.png)

This issue doesn't need immediate attention. I just created it to make sure its in the backlog.

**Comment by j0hnmeow** [comment]
It is much easier to just make a PR to the original repo than rewrite one of given plugins to `.tmTheme` file

**Comment by hertg** [comment]
> It is much easier to just make a PR to the original repo than rewrite one of given plugins to `.tmTheme` file

I agree, but as I've said in my post: The repository seems unmaintained and therefore very unlikely to accept a PR.
There are three open PRs from 2016 / 2017 and no closed ones, meaning there never even was a PR that made it into master.

![image](https://user-images.githubusercontent.com/3533648/95451602-514a3a80-0968-11eb-814c-c3d652cc05a1.png)

**Comment by j0hnmeow** [comment]
Then it's possible to make a fork and switch bat to using it. Unmaintenance is bad, but you're not forced to use the original repo.

**Comment by hertg** [comment]
> Then it's possible to make a fork and switch bat to using it. Unmaitenance is bad, but you're not forced to use the original repo.

Correct, but as sharkdp already pointed out in a comment on the PR #1229, he doesn't like to maintain such a fork himself, and neither do I, as I'm not familiar with the project.

I've already updated my initial post and mentioned existing forks of the repository that seem to be more up-to-date, maybe you missed that. But neither of them made any statement in the README about if those are just personal forks or if they are actively trying to maintain a stable continuation of the project.

**Comment by hertg** [comment]
**Update**

I just [commented on an issue](https://github.com/asciidoctor/sublimetext-asciidoc/issues/19#issuecomment-705524610) in the original repository and asked both people that are maintaining the forks if any of them would be willing to continue the project on their fork.

Hopefully we'll hear back from them. :)

**Comment by sharkdp** [comment]
Any update on this?
